btc-address-generation/src/main/kotlin/generation/btc/init/BtcAddressGenerationInitialization.kt

```diff
@@ -91,8 +94,12 @@ class BtcAddressGenerationInitialization(
             sessionAccount,
             btcAddressGenerationConfig.registrationAccount.accountId
         ).flatMap { details ->
+            // Getting time
+            val time = details[ADDRESS_GENERATION_TIME_KEY]!!.toLong()
+            // Removing generation time. Only public keys remain
+            details.remove(ADDRESS_GENERATION_TIME_KEY)
```

Comment:
We can rewrite it without line 98 since `remove`: `@return the previous value associated with the key, or `null` if the key was not present in the map.`